spec/support/helpers.rb

```diff
@@ -22,4 +22,60 @@ def destroy_repos
   def extract_text(string)
     Nokogiri::HTML(string).text.strip.gsub(/\s+/, ' ')
   end
+
+  def submit_files_before_due_date
+    pretend_now_is(Time.parse('July 20 2009 5:00PM')) do
```

Comment:
Do not use hardcoded dates (or hardcoded anything really) in helper functions.  Consider removing this function and using `submit_file_at_time` instead (below)